### ISSUE TITLE
ProceduralAnimation - not in master branch?

### ISSUE THREAD

**Issue by geslotencirkel** [keijiro/ProceduralMotionTrack#4]
Wondering if there are any issues for the very good animation component not being in master branch. Maybe something to watch out for? Works pretty well here and seems like an essential object for Unity (used it to fake realtime lights emitted from Spout since no RTX). Would be cool if animated texture could be linked to area light color but whatever.

**Comment by keijiro** [comment]
Sorry, I couldn't understand what you meant. Could you rephrase it?

- What is "master branch"? Usually it means the master branch of this repository (ProceduralMotionTrack).
- I have no idea about this sentence: "Would be cool if animated texture could be linked to area light color but whatever." It sounds like an irrelevant topic.

Basically, I can't speak/listen/read/write English. I spent lots of time to decode it every time. I'd appreciate if you could use simple plain English to explain your issues.

**Comment by geslotencirkel** [comment]
申し訳ありませんが、ライブビデオの色を使用してリアルタイムライトの色を変更するというランダムなアイデアは無視してください。

このリポジトリは手続き型モーション用ですが、手続き型アニメーションはここhttps://github.com/keijiro/ProceduralMotionTrack/tree/generic-propertyで非表示になっています。 なぜコードがここにないのか不思議に思っていますhttps://github.com/keijiro/ProceduralMotionTrack/tree/master。 多分何かおかしいですか？

**Comment by keijiro** [comment]
I tried to add generic property support in the generic-property branch, but I realized that it was a quite time-consuming task to implement it correctly. That branch has several bugs and design flaws, and I can't afford time for it.

I think this package is useful even without the generic property support, so I abandoned the idea and decided to keep maintaining the old implementation.

**Comment by keijiro** [comment]
I'm closing this issue now. Please feel free to reopen it for further problems.

**Comment by geslotencirkel** [comment]
Thank you for the clarification. Just to comment - it works very well for me and is fantastic. I simply merged it with the files in the Master branch and had no issues. Wish I found it sooner.
